website/projects/map4sci/src/app/shared/components/network/network.component.ts

```diff
@@ -130,21 +130,24 @@ export class NetworkComponent implements OnInit, OnChanges, OnDestroy {
           'height': 2000,
           'width': 2000,
           'backgroundColor': 'black',
-          'color': 'red'
+          'color': 'black',
+          'text-background-color': '#c4c7d6',
+          'text-background-opacity': 1,
```

Comment:
Try with a lower background opacity. If that works you can probably use a more 'traditional' white background with 0.5 (or so) transparency.